Automattic/jetpack
# Change WordAds and Simple Payments upsell CTAs to JP Creator LP

## Proposed changes:
Update upgrade buttons on Settings -> Earn page for WordAds and Simple Payments to point to JP Creator LP (jetpack.com/creator)

## Jetpack product discussion
https://github.com/Automattic/jetpack-marketing/issues/213

## Testing instructions:






* Apply the branch via Jetpack Beta on Jurassic Ninja site
* Go to Jetpack -> Settings -> Monetize
* Verify that upsell buttons point to jetpack.com/creator (page not published yet)

EDIT: More instructions
* Purchase Jetpack Creator for the website (you might need to do it with https://github.com/Automattic/wp-calypso/pull/83659
* Go to Jetpack > Dashboard > My Plan
* You should see the screen below (the Jetpack icon is fine for now):
![CleanShot 2023-11-02 at 12 39 18@2x](https://github.com/Automattic/jetpack/assets/8419292/b7bc7b0d-474a-4f9f-9b64-d776c87b6d70)
